packages/data-point/README.md

```diff
@@ -316,7 +316,7 @@ This method will return a **Promise** if `done` is omitted.
 | *options* | [Reducer](#reducers) | Request options. See this [example](#options-with-constants) for using constants in the reducer |
 | *done* | `function` _(optional)_ | Error-first callback [Node.js style callback](https://nodejs.org/api/errors.html#errors_node_js_style_callbacks) that has the arguments `(error, result)`, where `result` contains the final resolved [Accumulator](#accumulator). The actual transformation result will be inside the `result.value` property. |
 
-**<a name="transform-options">TransformOptions</a>**
+**<a name="transform-options">Options</a>**
```

Comment:
It's not visible, but do we also want to change the `name`?

Reply:
I could change it, just not sure what a better name would be (since `options` is probably too generic)